module/VuFind/src/VuFind/Db/Entity/PaymentFee.php

```diff
@@ -0,0 +1,370 @@
+    protected $currency;
+
+    /**
+     * Fine ID.
+     *
+     * @var string
+     */
+    #[ORM\Column(name: 'fine_id', type: 'string', length: 1024, nullable: false, options: ['default' => ''])]
+    protected $fineId;
+
+    /**
+     * Organization.
+     *
+     * @var string
+     */
+    #[ORM\Column(name: 'organization', type: 'string', length: 255, nullable: false, options: ['default' => ''])]
+    protected $organization;
+
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int
+    {
+        return $this->id ?? null;
+    }
+
+    /**
+     * Get payment.
+     *
+     * @return PaymentEntityInterface
+     */
+    public function getPayment(): PaymentEntityInterface
+    {
+        return $this->payment;
+    }
+
+    /**
+     * Set payment.
+     *
+     * @param PaymentEntityInterface $payment Payment.
+     *
+     * @return static
+     */
+    public function setPayment(PaymentEntityInterface $payment): static
+    {
+        $this->payment = $payment;
+        return $this;
+    }
+
+    /**
+     * Get title.
+     *
+     * @return string
+     */
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    /**
+     * Set title.
+     *
+     * @param string $title Title
+     *
+     * @return static
+     */
+    public function setTitle(string $title): static
+    {
+        $this->title = mb_substr($title, 0, 255, 'UTF-8');
+        return $this;
+    }
+
+    /**
+     * Get type.
+     *
+     * @return string
+     */
+    public function getType(): string
+    {
+        return $this->type;
+    }
+
+    /**
+     * Set type.
+     *
+     * @param string $type Type
+     *
+     * @return static
+     */
+    public function setType(string $type): static
+    {
+        $this->type = mb_substr($type, 0, 255, 'UTF-8');
+        return $this;
+    }
+
+    /**
+     * Get description.
+     *
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
+
+    /**
+     * Set description.
+     *
+     * @param string $description Description
+     *
+     * @return static
+     */
+    public function setDescription(string $description): static
+    {
+        $this->description = mb_substr($description, 0, 255, 'UTF-8');
+        return $this;
+    }
+
+    /**
+     * Get amount (in pennies, including any tax).
+     *
+     * @return int
+     */
+    public function getAmount(): int
+    {
+        return $this->amount;
+    }
+
+    /**
+     * Set amount (in pennies, including any tax).
+     *
+     * @param int $amount Amount
+     *
+     * @return static
+     */
+    public function setAmount(int $amount): static
+    {
+        $this->amount = $amount;
+        return $this;
+    }
+
+    /**
+     * Get tax percent (in 1/100ths or a percent).
```

Comment:
Typo:

```suggestion
     * Get tax percent (in 1/100ths of a percent).
```

...and note that this "or a percent" typo is repeated in multiple places, so you might want to do a search-and-replace to fix them all.

Reply:
Oops, fixed.